server/protocols/webadmin/README.adoc

```diff
@@ -136,10 +136,17 @@ Resource name mailboxNameToBeCreated should not be empty
 
 Response codes :
  - 204 : The mailbox now exists on the server
- - 404 : The user name does not exist
+ - 400 : The user name does not exist
```

Comment:
I don't think you need to change this case.
I asked for a 400 on bad input actually.